pages/docs/advanced/react-native.en-US.mdx

```diff
@@ -96,3 +96,64 @@ import { AppState } from 'react-native'
 ```
 
 For `initReconnect`, it requires some 3rd party libraries such as [NetInfo](https://github.com/react-native-netinfo/react-native-netinfo) to subscribe to the network status. The implementation will be similar to the example above: receiving a `callback` function and trigger it when the network recovers from offline, so SWR can start a revalidation to keep your data up-to-date.
+
+
+### AsyncStorage Based Persistent Cache
+
+You might want to sync your cache to `asyncStorage`. Here's an example implementation:
```

Comment:
```suggestion
You might want to sync your cache to `AsyncStorage`. Here's an example implementation:
```

Reply:
Thanks for the feedback, I'm gonna update this one @huozhi